src/components/modal/modal.module.scss

```diff
@@ -0,0 +1,80 @@
+@use "../../styles/abstracts/index" as *;
+
+.dialogOverlay {
+  position: fixed;
+  inset: 0;
+  background-color: var(--color-dark-700);
```

Comment:
Please double check color, I guess here should be rgba format with some transparency

Reply:
change color on "rgb(0 0 0 / 60%)"